stack/rest/src/main/java/org/apache/usergrid/rest/management/ManagementResource.java

```diff
@@ -183,6 +201,11 @@ private Response getAccessTokenInternal( UriInfo ui, String authorization, Strin
             String errorDescription = "invalid username or password";
 
             if ( user == null ) {
+
+                // make sure authentication is allowed considering
+                // external token validation configuration (UG Central SSO)
+                ensureAuthenticationAllowed( username, grant_type );
```

Comment:
I moved the "superuser-only login when external token validation enabled" logic into a new method called ensureAuthenticationAllowed()